misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -68,13 +69,10 @@ class AwsSqsJobQueueModule(
   }
 
   @Provides @ForSqsConsumer @Singleton
-  fun consumerRepeatedTaskQueue(clock: Clock, config: AwsSqsJobQueueConfig): RepeatedTaskQueue {
-    return RepeatedTaskQueue(
+  fun consumerRepeatedTaskQueue(queueFactory: RepeatedTaskQueueFactory,
+    config: AwsSqsJobQueueConfig): RepeatedTaskQueue {
+    return queueFactory.new(
         "sqs-consumer-poller",
-        clock,
-        Executors.newCachedThreadPool(ThreadFactoryBuilder()
-            .setNameFormat("sqs-consumer-%d")
```

Comment:
the name of the thread is going to change to `sqs-consumer-poller-%d`. shouldn't be a big deal